### ISSUE TITLE
[BUG] nz-dropdown 指定方向后，sub-menu 方向不会改变

### ISSUE THREAD

**Issue by cipchk** [NG-ZORRO/ng-zorro-antd#84]
## I'm submitting a...

<pre><code>
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Support request => Please do not submit support request here
</code></pre>

## Current behavior

`nz-dropdown` 指定方向了 `bottomRight`，当在最右边的情况下，一级 `nz-menu` **正常**，而 `nz-submenu` 的方向无法设置，倒置子菜单被隐藏。

```html
<nz-dropdown nzTrigger="click" nzPlacement="bottomRight">
    <div class="item" nz-dropdown>
        <i class="anticon anticon-setting"></i>
    </div>
    <ul nz-menu class="pt-sm pb-sm wd-sd">
        <li nz-submenu>
            <span title>
                <i class="anticon anticon-edit"></i>
                Language
            </span>
            <ul>
                <li nz-menu-item *ngFor="let item of tsServ.langs" [nzSelected]="item.code === settings.layout.lang">{{item.text}}</li>
            </ul>
        </li>
    </ul>
</nz-dropdown>
```

![tim 20170818155618](https://user-images.githubusercontent.com/2987467/29449553-c857917c-842d-11e7-8095-7876b0ca6e76.png)

宽松上来讲这不算是一个着急的BUG；是否有一些其他解决方案帮助及时处理：最靠右边的情况下，我如何展开我的二级菜单？

**Comment by vthinkxie** [comment]
sub menu auto position not supported yet.

**Comment by vthinkxie** [comment]
related https://github.com/ant-design/ant-design/issues/2830

**Comment by vthinkxie** [comment]
fixed in version 0.7.0
